index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <title>Benjamin-Portfolio</title>
+  </head>
+
+  <body>
+    <main class="main ">
+      <div class="nav-flex ">
+      
+        <header>
+      
+          <nav class="main-nav">
```

Comment:
- [ ] kindly align the logo and the hamburger icon well
![Capturefx](https://user-images.githubusercontent.com/81444338/170700379-7233a198-3a2a-441d-8d1d-2606ae7dac2e.PNG)